smokey/src/main/java/frc/robot/commands/AutoBalanceStepCommand.java

```diff
@@ -24,8 +26,11 @@ public class AutoBalanceStepCommand extends CommandBase{
   private double waitDurationSecondsValue = .7;
   // TODO hardcoded values eventually replaced when we implement PID controller drive command
   private double driveDurationSecondsValue = 0.2;  
-  private double velocityValue = 0.6;
+  private double velocityValue = 0.9;
   private int numIterations = 0;
+  // ramp max slope = 15 degrees
+  // use velocity value for errors >= sin(10) degrees, and proprortaional for smaller errors
+  private PIDController pidController = new PIDController(velocityValue/Math.sin(Math.toRadians(10)),0.0,0.0);
```

Comment:
just the 10 degrees magic number not matching the 15 degree ramp max slope comment above ... presumably 10 is right but not super clear on why 10 and 15 are mentioned

Reply:
added more commentary to explain.